src/bin/rdcore/cmdline.rs

```diff
@@ -75,6 +83,45 @@ pub fn parse_args() -> Result<Config> {
                         .takes_value(true),
                 ),
         )
+        .subcommand(
+            SubCommand::with_name("kargs")
```

Comment:
WDYT about also exposing an argument for just fetching the kargs from the BLS config? That way we can keep all karg interactions (read, append, delete) inside of calls to `rdcore kargs`.

Reply:
`rdcore kargs` now always prints out the final kargs (after optionally modifying it).